x-pack/plugins/transform/public/app/sections/transform_management/components/action_reset/reset_action_name.tsx

```diff
@@ -24,45 +27,78 @@ export const resetActionNameText = i18n.translate(
 );
 
 const transformCanNotBeReseted = (i: TransformListRow) =>
+  i.stats &&
   !([TRANSFORM_STATE.STOPPED, TRANSFORM_STATE.FAILED] as TransformState[]).includes(i.stats.state);
 
 export const isResetActionDisabled = (items: TransformListRow[], forceDisable: boolean) => {
   const disabled = items.some(transformCanNotBeReseted);
-  return forceDisable === true || disabled;
+  const hasNoStats = items.some((i: TransformListRow) => !isTransformListRowWithStats(i));
+
+  return forceDisable === true || disabled || hasNoStats;
 };
 
+export const getResetActionDisabledMessage = ({
+  items,
+  canResetTransform,
+  forceDisable,
+}: {
+  items: TransformListRow[];
+  canResetTransform: TransformCapabilities['canResetTransform'];
+  forceDisable: boolean;
+}) => {
+  const isBulkAction = items.length > 1;
+
+  // Disable start for transforms if stats does not exist
```

Comment:
Should be `Disable reset...`

Reply:
Removed here [`82de19d` (#166320)](https://github.com/elastic/kibana/pull/166320/commits/82de19d62bea414640929fd4b106c105a51e1a53)